frontend/pages/ManageControlsPage/OSSettings/cards/DiskEncryption/components/DiskEncryptionTable/DiskEncryptionTableConfig.tsx

```diff
@@ -139,7 +129,7 @@ const windowsTableHeader: IDataColumn[] = [
 ];
 
 export const generateTableHeaders = (): IDataColumn[] => {
```

Comment:
I think we can just return `defaultTableHeaders` here instead of the copy. Or just export the const itself and remove the calls `generateTableHeaders`. I like keeping a stable array reference where possible is always a good idea when it is being used in downstream props, especially with components like the table table that can be render intensive. (This disk encryption table is a pretty light use of react-table so it isn't a big concern in this case, more of a general observation).

Reply:
Good point!